src/globals.test.js

```diff
@@ -445,4 +453,26 @@ describe('globals', () => {
 
     process.env = { ...oldEnv };
   });
+
+  test.each`
```

Comment:
It looks like its hard to maintain these spaces, maybe array its better

Reply:
tried both versions,  decided to stay with the table version